Code/Framework/AzCore/AzCore/Component/EntitySerializer.h

```diff
@@ -30,4 +30,23 @@ namespace AZ
         void ConvertComponentVectorToMap(const AZ::Entity::ComponentArrayType& components,
             AZStd::unordered_map<AZStd::string, AZ::Component*>& componentMapOut);
     };
+
+    /**
+     * Helper class to track components that have been skipped during loading.
+     * When this class is added to the metadata of a Json deserializer setting,
+     * custom component serializers can add themselves to the list so users
+     * can be informed of component deprecation upon load completion
+     */
```

Comment:
nit: It's generally preferred to use the `//!` style of comments now as they're easier to maintain, so this would just become...

```
//! Helper class to track components that have been skipped during loading.
//! When this class is added to the metadata of a Json deserializer setting,
//! custom component serializers can add themselves to the list so users
//! can be informed of component deprecation upon load completion
```